beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -2953,49 +2953,6 @@ func TestProposer_DeleteAttsInPool_Aggregated(t *testing.T) {
 	assert.Equal(t, 0, len(atts), "Did not delete unaggregated attestation")
 }
 
-func TestProposer_GetSyncAggregate_OK(t *testing.T) {
```

Comment:
I moved this test to `proposer_altair_test.go` so that all sync committee tests are in the same file